packages/@aws-cdk/aws-ssm/lib/parameter.ts

```diff
@@ -161,6 +168,13 @@ export interface StringParameterAttributes {
    * @default The latest version will be retrieved.
    */
   readonly version?: number;
+
+  /**
+   * The type of the string parameter
+   *
+   * @default "String"
+   */
+  readonly type?: string;
```

Comment:
Maybe an enum?

Reply:
Yeah I was thinking about it, but it would have required refactoring and exporting these constants into an enum: https://github.com/aws/aws-cdk/blob/b996cf5753767c86bef595f334d893964f7133ed/packages/%40aws-cdk/aws-ssm/lib/parameter.ts#L155-L157

Is that alright with you?

Reply:
I think that'd be great.

Reply:
I would expect to be writing something like `ParameterType.STRING`.

Reply:
Done